### ISSUE TITLE
support more keybindings for scrolling

### ISSUE THREAD

**Issue by dpsutton** [vlaaad/reveal#29]
Love this amazing project. Was wondering if you would be open to more keybindings for scrolling. Perhaps as a config, or with such a sparse keymap so far, maybe both enabled since they don't seem to overlap:

- vim: h, j, k, l
- emacs: n, p, f, b

**Comment by vlaaad** [comment]
Hmm, I would expect those to move the cursor instead of scrolling.. or did you mean moving the cursor?

**Comment by vlaaad** [comment]
Btw I want to implement configurable keybindings system, but currently I'm designing spec forms, so that will come some time later

**Comment by dpsutton** [comment]
Yes sorry i meant for moving the cursor. Each of vim and emacs have their own notion of selection (visual mode and mark) but perhaps just continuing to keep shift and cursor movements would be a happy medium for everyone.

Glad its on the horizon and thanks!
